app/models/edition.rb

```diff
@@ -207,10 +207,25 @@ def self.to_date(date)
   end
 
   def self.only_broken_links
-    joins("INNER JOIN link_checker_api_reports ON link_checker_api_reports.link_reportable_id = editions.id")
-    .joins("INNER JOIN link_checker_api_report_links ON link_checker_api_report_links.link_checker_api_report_id = link_checker_api_reports.id")
-    .where("link_checker_api_reports.link_reportable_type = 'Edition' AND link_checker_api_report_links.status != 'ok'")
```

Comment:
I _think_ you could accomplish this with an additional `where` rather than a full rewrite:

```suggestion
    .where("link_checker_api_reports.id = (SELECT MAX(id) FROM link_checker_api_reports WHERE link_checker_api_reports. link_reportable_id = editions.id)")`
    .where("link_checker_api_reports.link_reportable_type = 'Edition' AND link_checker_api_report_links.status != 'ok'")`
```

Reply:
Yeah, that looks like it might work, although I think the `GROUP BY` approach is a little neater.